app/styles/app.scss

```diff
@@ -218,7 +218,7 @@ article.aria-roles {
     margin-bottom: 0.5em;
   }
 .demo-div {
-  flex: 1 1 1;
```

Comment:
we might want to consider using something other than flexbox for browser support.